test/functional/apps/dashboard/dashboard_state.js

```diff
@@ -277,14 +277,14 @@ export default function ({ getService, getPageObjects }) {
 
           await retry.try(async () => {
             const pieSliceStyle = await pieChart.getPieSliceStyle('80,000');
-            // The default green color that was stored with the visualization before any dashboard overrides.
-            expect(pieSliceStyle.indexOf('rgb(87, 193, 123)')).to.be.greaterThan(0);
+            // The default color that was stored with the visualization before any dashboard overrides.
+            expect(pieSliceStyle.indexOf('rgb(84, 179, 153)')).to.be.greaterThan(0);
           });
         });
 
         it('resets the legend color as well', async function () {
           await retry.try(async () => {
-            const colorExists = await PageObjects.visChart.doesSelectedLegendColorExist('#57c17b');
+            const colorExists = await PageObjects.visChart.doesSelectedLegendColorExist('#54B399');
```

Comment:
@manasvinibs Can you open a follow-up issue to refactor/improve these tests to be less brittle?

Reply:
oh yes I already have the issue for that https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4424